lib/galaxy/datatypes/binary.py

```diff
@@ -4747,3 +4747,8 @@ def display_peek(self, dataset: DatasetProtocol) -> str:
             return dataset.peek
         except Exception:
             return f"Binary numpy file ({nice_size(dataset.get_size())})"
+
+class bcsp(Binary):
```

Comment:
You don't need to do that.

Reply:
So i can delete this?

Reply:
https://github.com/galaxyproject/galaxy/pull/19363/commits/9ead291dadd50c3240d0b773156eb88c15a19fd8